tests/unittests/test_wsgi.py

```diff
@@ -30,9 +30,10 @@
 # =================================================================
 """Unit tests for pycsw.wsgi"""
 
+from unittest import mock
+
```

Comment:
please remove this blank line, as wsgiref is also part of the Python stdlib

Reply:
thanks for review